File: event-bus-rabbit/src/main/dotnet/topology/events/TopologyUpdate.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;


namespace pegasus.eventbus.topology.events
{
	public class TopologyUpdate
	{
        public TopologyRegistry TopologyRegistry;


        public TopologyUpdate() { }

        public TopologyUpdate(TopologyRegistry topoRegistry) { this.TopologyRegistry = topoRegistry; }
	}
}

File: event-bus-rabbit/src/main/java/pegasus/eventbus/topology/events/TopologyUpdate.java
package pegasus.eventbus.topology.events;

import pegasus.eventbus.topology.TopologyRegistry;

public class TopologyUpdate {

    private TopologyRegistry topologyRegistry;

    public TopologyRegistry getTopologyRegistry() {
        return topologyRegistry;
    }

    public void setTopologyRegistry(TopologyRegistry topologyRegistry) {
        this.topologyRegistry = topologyRegistry;
    }

}
